tests/utils/test_ellipsoid_utils.py

```diff
@@ -0,0 +1,84 @@
+"""Unit tests for functions in ellipsoid utils file.
+
+Authors: Adi Singh
+"""
+import gtsfm.utils.ellipsoid as ellipsoid_utils
+import numpy as np
+import numpy.testing as npt
+import unittest
+
+
+class TestEllipsoidUtils(unittest.TestCase):
+    """Class containing all unit tests for ellipsoid utils."""
+
+    def test_center_point_cloud(self):
+        """Tests the center_point_cloud() function with 5 sample points."""
+
+        sample_points = np.array([[6, 13, 9], [5, 9, 7], [6, 10, 9], [9, 13, 6], [5, 12, 9]])
```

Comment:
could probably choose even simpler points, and add an inline comment showing the mental math

like just 3 points